src/js/index.js

```diff
@@ -20,43 +20,41 @@ export function getElectronegativity(symbol) {
     return config ? config["pauling_negativity"] : 0;  // return zero if value cannot be accessed by symbol
 }
 
+export function filterBondsDataByElementsAndOrder(bondsData, element1, element2, order = undefined) {
```

Comment:
Let's have the docstring above and properly format the ones below too - with @param, @return etc

Reply:
Done.